### PR TITLE
Quality data improvements

### PR DESCRIPTION
**Pull request by michaelwood**

### CODE REVIEW

**Comment by michaelwood** [comment]
Depends on : https://github.com/ThreeSixtyGiving/dataquality/pull/95

File: datastore/data_quality/quality_data.py [hunk 1]
```diff
@@ -357,12 +363,8 @@ def get_publisher_quality_grants(self):
                 continue
 
             # For compatibility badge value 100 = True , 0 = False
-            ret[metric] = (
-                100
-                if self.source_file_set.count()
-                == self.source_file_set.filter(**query).count()
-                else 0
-            )
+            ret[metric] = 100 if self.source_file_set.filter(**query).count() > 0 else 0
```
Comment (R2ZER0) [inline datastore/data_quality/quality_data.py:366]: Am I understanding this right: The metric changed from "all source files must match the condition" to "at least one source file must match the condition"?

Comment (michaelwood) [reply datastore/data_quality/quality_data.py:366]: Yes that's correct.

Comment (michaelwood) [reply datastore/data_quality/quality_data.py:366]: Now that you've reminded me of that I think that could do with being in the commit message.

Comment (michaelwood) [reply datastore/data_quality/quality_data.py:366]: Updated commit message

**Review by R2ZER0 — APPROVED** [review]
Looks good :+1:

**Pull request merged**